core/capabilities/integration_tests/streams_test.go

```diff
@@ -45,8 +45,8 @@ func Test_OneAtATimeTransmissionSchedule(t *testing.T) {
 
 	// The don IDs set in the below calls are inferred from the order in which the dons are added to the capabilities registry
 	// in the setupCapabilitiesRegistryContract function, should this order change the don IDs will need updating.
-	workflowDonInfo := createDonInfo(t, don{id: 1, numNodes: 5, f: 1})
-	triggerDonInfo := createDonInfo(t, don{id: 2, numNodes: 7, f: 1})
```

Comment:
This was, technically, fine as N > 3F+1 (it only needs to be greater, not necessarily equal).